FHEM/lib/SD_ProtocolData.pm

```diff
@@ -1772,21 +1772,41 @@ package lib::SD_ProtocolData;
 							# dim100%   MU;P0=-10420;P1=-92;P2=398;P3=-417;P5=596;P6=-592;D=1232323232323232323232323562323235656232323232356232356232623232323232323232323232323235623232323562356565623565623562023232323232323232323232356232323565623232323235623235623232323232323232323232323232323562323232356235656562356562356202323232323232323;CP=2;R=72;
 			{
 				name				=> 'FS20',
-				comment				=> 'remote control',
+				comment				=> 'remote control (decode as MU)',
 				id				=> '74',
 				knownFreqs			=> '868.35',
 				one				=> [1.5,-1.5],	# 600
 				zero				=> [1,-1],	# 400
 				pause				=> [-25],
 				clockabs			=> 400,
-				#reconstructBit			=> '1',
+				reconstructBit			=> '1',
 				format				=> 'twostate',	# not used now
 				clientmodule			=> 'FS20',
 				preamble			=> '810b04f70101a001',
 				length_min			=> '50',
 				length_max			=> '67',
 				postDemodulation	=> \&main::SIGNALduino_postDemo_FS20,
 			},
+		"74.1"	=>	## FS20 - Remote Control (868Mhz) @HomeAutoUser
```

Comment:
Hat es einen Grund, dass du in der bestehenden ID 74 reconstructBit setzt und in der neuen ID 74.1 das nicht machst.
Mir ist bewusst, dass der Unterschied sich nur auf das reconstructBit bezieht, nur weiss ich nicht, wieso Du das nicht in 74.1 aktiviert hast und 74 dafür unangetastet lässt.

Reply:
reconstructBit war ein Kopierfehler, welcher durchs testen auf dem Testsystem noch gesetzt war.
Es kommt heraus wie bei 74. ;-) *thanks*